### PR TITLE
Watched components

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: weblate/accounts/models.py [hunk 1]
```diff
@@ -419,7 +419,7 @@ class Profile(models.Model):
     DASHBOARD_COMPONENT_LISTS = 6
 
     DASHBOARD_CHOICES = (
-        (DASHBOARD_WATCHED, _('Watched translations')),
+        (DASHBOARD_WATCHED, _('Watched components')),
```
Comment (nijel) [inline weblate/accounts/models.py]: Actually it's "Translations to your languages from watched components", so neither of these is accurate.

Comment (comradekingu) [reply weblate/accounts/models.py]: **Watched component string changes in your languages**
is not good
**New translations from watched components in your languages**
is somewhat better

Doesn't seem to upset anything length-wise

image

Comment (nijel) [reply weblate/accounts/models.py]: This is about *Dashboard* - what will be default tab shown there. The same string is used as label there.

**Comment by nijel** [comment]
I've merged some minor changes here, but the biggest change is IMHO wrong - this is about what is shown on dashboard, not about notifications. Also "Watched components" sounds wrong label for list of translations from watched components.

**Pull request closed without merging**